internal/cmd/tui.go

```diff
@@ -158,6 +160,7 @@ func tuiCmd() *cobra.Command {
 	cmd.Flags().BoolVar(&runOnce, "exit", false, "Exit TUI after running a command")
 	cmd.Flags().IntVar(&visibleEntries, "entries", defaultVisibleEntries, "Number of entries to show in TUI")
 	cmd.Flags().StringVar(&serverAddr, "server", os.Getenv("RUNME_SERVER_ADDR"), "Server address to connect TUI to")
+	cmd.Flags().BoolVar(&disableBackgroundProcesses, "disable-background", false, "Disable running background blocks as background processes")
```

Comment:
In a follow-up PR I'm gonna do a refactor so that there's less duplication of command flags between the `run` and `tui` commands.